crates/bitwarden/Cargo.toml

```
@@ -16,7 +16,10 @@ rust-version = "1.71"
 default = ["secrets"]
 
 secrets = [] # Secrets manager API
-internal = [] # Internal testing methods
+internal = [
+    "bitwarden-exporters",
+    "bitwarden-generators",
```

```suggestion
    "dep:bitwarden-exporters",
    "dep:bitwarden-generators",
```

If we don't use the `dep:` prefix, cargo tries to be helpful and expose a feature with the crate name, so you can do `cargo b -p bitwarden --features bitwarden-generators`

Using `dep:` removes the implicit feature